app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailsFragment.java

```diff
@@ -110,39 +85,13 @@ public void onDetach() {
 
     @Override
     public void showProgressBar(boolean show) {
-        ViewUtils.showView(progressBar, show);
-    }
-
-    @Override
-    public void showEventName(String name) {
-        tvEventTitle.setText(name);
-    }
-
-    @Override
-    public void showDates(String start, String end) {
-        tvStartDate.setText(start);
-        tvEndDate.setText(end);
+        ViewUtils.showView(binding.progressBar, show);
```

Comment:
Yes. This is a proposed feature, where we will set a state to the model and change it to reflect in the binding, but currently we are changing the objects so a new object will override the state instead of toggling it. This is something we can work in future with better design on how to handle state changes